spec/access-identify.md

```diff
@@ -0,0 +1,303 @@
+## `access/identify`
+
+Upload protocol defines [access/identify][] capability which is very flexible yet powerful so it can be very confusing. This document attempts to wolk through some user stories to try and clarify.
+
+#### identify with email
+
+"Accounting" service MAY delegate other "verifier" service(s) to do out of bound user verification e.g. sign-in via email link style flow.
```

Comment:
```suggestion
"Accounting" service MAY delegate to other "verifier" service(s) to do out of band user verification e.g. sign-in via email link style flow.
```

What does "Accounting" mean here? is it a name for "a service that does a thing e.g. and Accounting app" or does it have more meaning?